src/datastore_mad/remotes/lxd_downloader.sh

```diff
@@ -169,6 +169,10 @@ export PATH=\$PATH:/bin:/sbin
 
 echo "nameserver $DNS_SERVER" > /etc/resolv.conf
 
+yum install epel-release openssh-server -y > /dev/null 2>&1
+/usr/bin/echo "centos6" >> /root/version
```

Comment:
What is the purpose of these `echo centos6` and 7?

Reply:
ups, missed those debug lines